src/hooks/index.js

```diff
@@ -493,20 +518,25 @@ exports.sendEmail = (hook, maildata) => {
 		});
 	}
 	else {
-		_.uniq(receipients).map(email=> {
-			mailService.create({
-				email: email,
-				subject: maildata.subject || "E-Mail von der Schul-Cloud",
-				headers: maildata.headers || {},
-				content: {
-					"text": maildata.content.text || { "text": "No alternative mailtext provided. Expected: HTML Template Mail." },
-					"html": ""
-				}
-			})
-			.catch (err => {
-				throw new errors.BadRequest((err.error||{}).message || err.message || err || "Unknown mailing error");
+		if (!maildata.content.text && !maildata.content.html) {
```

Comment:
Can this code block be merged with the above? It seems to be the same code to me